parser/src/parser/grammar.lalrpop

```diff
@@ -118,6 +119,24 @@ PeriodicColumn: PeriodicColumn = {
         PeriodicColumn::new(name, values),
 }
 
+// RANDOM VALUES
+// ================================================================================================
+
+RandomValues: RandomValues = {
+    "random_values" ":" <rand_values: RandValues> => rand_values
+}
+
+RandValues: RandomValues = {
+    <name: Identifier> ":" <index: Index> => RandomValues::new(name, index, vec![]),
+    <name: Identifier> ":" "[" "]" => RandomValues::new(name, 0, vec![]),
+    <name: Identifier> ":" "[" <rand_vec: CommaElems<RandElem>> "]" => RandomValues::new(name, rand_vec.iter().map(|v| v.size()).sum::<usize>(), rand_vec)
+}
```

Comment:
I think these could be named more clearly. One of them is the section, while the other one is the declaration. I know it fits with our existing convention to call the source section one `RandomValues`, but maybe the second could be `RandValuesDecl` or something like  that?